addons/l10n_in_edi/models/account_edi_format.py

```diff
@@ -29,13 +29,26 @@ def _is_enabled_by_default_on_journal(self, journal):
             return journal.company_id.country_id.code == 'IN'
         return super()._is_enabled_by_default_on_journal(journal)
 
+    def _get_l10n_in_base_tags(self):
+        return (
+           self.env.ref('l10n_in.tax_tag_base_sgst').ids
+           + self.env.ref('l10n_in.tax_tag_base_cgst').ids
+           + self.env.ref('l10n_in.tax_tag_base_igst').ids
+           + self.env.ref('l10n_in.tax_tag_base_cess').ids
+           + self.env.ref('l10n_in.tax_tag_zero_rated').ids
+           + self.env.ref("l10n_in.tax_tag_exempt").ids
+           + self.env.ref("l10n_in.tax_tag_nil_rated").ids
+           + self.env.ref("l10n_in.tax_tag_non_gst_supplies").ids
+        )
+
     def _get_move_applicability(self, move):
         # EXTENDS account_edi
         self.ensure_one()
         if self.code != 'in_einvoice_1_03':
             return super()._get_move_applicability(move)
-
-        if move.is_sale_document() and move.country_code == 'IN' and move.l10n_in_gst_treatment in (
+        all_base_tags = self._get_l10n_in_base_tags()
+        is_under_gst = any(move_line_tag.id in all_base_tags for move_line_tag in move.line_ids.tax_tag_ids)
+        if move.is_sale_document(include_receipts=True) and move.country_code == 'IN' and is_under_gst and move.l10n_in_gst_treatment in (
```

Comment:
@jva-odoo To understand better: is there a GST treatment where we would not send it and there would still be GST taxes?  And there are cases of e.g. GST treatment regular where we do not send it at all?